### PR TITLE
Fixes and Enhancements

### PR DESCRIPTION
**Pull request by brettwinters**
I'm new to JS/Jquery, but I created a loop to allow multiple tables to be painted using the $(".table").treetable(); selector. The previous master got confused if multiple tables were selected on the same page

Also fixed a problem where calling the plugin repeatedly would re-paint the content. I propose using the tt-table class to decide whether the tt-table has already been painted. 

Also, for people using the tt-table in tabs, where the table is not initially visible, the element.position() method won't work (returns top=0, left = 0) so in these cases I propose skipping the draw of the content. Later on the ".on('shown.bs.tab', function (e) {" event we can redraw the tables again

### CODE REVIEW

**Review by martinsbalodis — CHANGES_REQUESTED** [review]
It seems that you have replaced tabs with spaces. This causes the diff tool think that you rewrote the entire file. Can you do another pull request that has tabs? https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Comment by brettwinters** [comment]
Oops. Not sure how that happened - let me try again....

________________________________
